web/services/clusters.go

```diff
@@ -58,7 +58,7 @@ func (s *clustersService) GetAll(filters map[string][]string) (models.ClusterLis
 		}
 	}
 
-	err := db.Find(&clusters).Error
+	err := db.Find(&clusters).Order("name").Order("id").Error
```

Comment:
Hey @fabriziosestito ,
Just to be sure. Did you test if this works?
When I played a bit with this, I couldn't get the correct ordering if I didn't put the `Order` clause before the find.
Like this:
```db.Order("name").Order("id").Find(&clusters).```

Just as a comment hehe

Reply:
Resolved thanks!